client/state/data-layer/wpcom/read/following/mine/test/utils.js

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
pure move. resolved a dependency cycle. following/mine/index requires following/mine/new/index, which also requires following/mine/index. the cycle doesn't break anything, but it _can_ make the build / static analysis grumpy. better not to have it.